its/src/test/java/com/sonar/it/jenkins/orchestrator/JenkinsOrchestrator.java

```diff
@@ -267,14 +270,9 @@ public JenkinsOrchestrator newFreestyleJobWithMaven(String jobName, File project
     setTextValue(findElement(By.name("_.targets")), "clean package");
 
     findElement(buttonByText("Add build step")).click();
-    
+
     findElement(By.linkText("Invoke top-level Maven targets")).click();
-    try {
-      Thread.sleep(2000);
-    } catch (InterruptedException e1) {
-      e1.printStackTrace();
-    }
-    setTextValue(findElement(By.name("_.targets"), 1), getMavenParams(orchestrator));
+    setTextValue(findElement(driver, By.name("_.targets"), 1), getMavenParams(orchestrator));
```

Comment:
FYI, the sleep was added as a dirty workaround for a timing issue in ITs executed by cix. Without this sleep, a window will pop-up asking if you want to navigate away from the page with unsaved changes. That said, the sleep was not intended as a solution, just an experiment to see if it indeed helps. It did seem to help, but then we didn't clean it up. If you remove this, probably the build will fail on cix, complaining about a pop-up.